Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -152,11 +139,42 @@ class JobManagerExample: TableViewController {
         
         // job cell registration
         tableView.register(JobTableViewCell.self, forCellReuseIdentifier: "JobCell")
+        
+        // resume any paused jobs
+        JobManager.shared.resumeAllPausedJobs(
```

Comment:
Now that there is a corresponding action when the view controller goes away, I'm wondering if a more appropriate place for resuming the jobs is `viewDidAppear(_:)` and the jobs should be paused in `viewWillDisappear(_:)`. That way any authentication challenges are only shown when the view controller is on screen.

Reply:
that might be a better option. I can change to that.